polar-core/src/polar.lalrpop

```diff
@@ -126,6 +142,30 @@ Call: Value = {
     }
 };
 
+DotCall: Value = {
```

Comment:
I think we could probably reduce the duplication between `Call` and `DotCall` with a generic param, but I'm not sure it's worth the effort. Up to you whether you want to experiment with that or just ship it.

Reply:
Good call, I'll revisit this later, not familiar enough with lalrpop to know how to do it right now.